src/tarkov-data-manager/jobs/archive-prices.mjs

```diff
@@ -23,50 +23,53 @@ class ArchivePricesJob extends DataJob {
 
         // archive max_days_per_run number of days
         for (let i = 0; i < max_days_per_run; i++) {
-            // get the price with the oldest timestamp
-            const oldestPrice = await this.query(`
-                SELECT * FROM price_data 
-                WHERE timestamp < ?
-                ORDER BY timestamp
-                LIMIT 1
-            `, [cutoff]);
-            if (oldestPrice.length === 0) {
-                this.logger.success(`No prices found before ${cutoff}`);
-                return;
+            for (let pve = 0; pve < 2; pve++) {
```

Comment:
Seemed like the simplest way to not duplicate code lol

Reply:
You can refactor the whole pve loop as a function and pass pve to it as a parameter. (but I guess there are other factors why this can't be done easily that I don't see by just looking at the code)